WTraining/app/services/weet.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+const { requestGet } = require('../helpers/request');
+const logger = require('../logger');
+const config = require('../../config').common.url;
+
+exports.getWeet = () =>
+  requestGet(config.urlWeet)
+    .then(res => res.data)
+    .catch(e => {
+      logger.error(e);
+      throw e;
```

Comment:
You could to throw a specific error using the Wolox error definition. I think that You should created a error in `app/errors.js `and then to define code status error in `app/middleware/errors.js ` for example 

`errors.js `
` exports.VALIDATION_ERROR = 'validation_error';
  exports.validationError = message => internalError(message, exports.VALIDATION_ERROR);
`

`middleware/errors.js
`
`
const statusCodes = {
  [errors.VALIDATION_ERROR]: 422
};
`